.github/actions/report-job-status/README.md

```diff
@@ -0,0 +1,31 @@
+# Send VK Teams notification on failure
+
+This action selects the corresponding chat for the notification on failure and
+uses [report-job-status action](https://github.com/tarantool/actions/tree/master/report-job-status)
+for sending it to:
+* the team chat (Tarantool CI/CD reports) - if the job started on any event in
+the `master` or release branch (`1.10`, `2.10`, etc);
+* a personal chat, created by the committer – if the job started on
+creating/updating a pull request or any event in other branches.
+
+## How to create personal chat
+
+1. Create a new public chat in VK Teams (access can be limited by enabling
+`Join in Approval` in the chat settings).
+2. Change URL of the created chat: use URL of the team chat and add
+`_<your_github_nickname>` to it, for example:
+`https://vkteams.example.com/profile/team_chat_mynickname`.
+3. Add the bot from the team chat to your personal chat.
```

Comment:
I think it is worth mentioning the user nickname:

Add the bot user `tarantoolbot` to your personal chat.

Reply:
Thanks, looks like fixed.